apps/content_types/content_types.py

```diff
@@ -188,8 +188,11 @@ def prepare_for_edit_content_type(doc):
 def init_extra_fields(editor, schema):
     fields = get_resource_service('vocabularies').get_extra_fields()
     for field in fields:
+        field_type = field.get('field_type', None)
```

Comment:
there should be some default value, otherwise the type would be left empty, while before it was always set